base/applications/regedit/main.c

```diff
@@ -45,12 +43,12 @@ BOOL InitInstance(HINSTANCE hInstance, int nCmdShow)
     HMENU hEditMenu;
     INITCOMMONCONTROLSEX icce;
     WNDCLASSEXW wcFrame;
-    WNDCLASSEXW wcChild;
     ATOM hFrameWndClass;
 
     ZeroMemory(&wcFrame, sizeof(WNDCLASSEXW));
     wcFrame.cbSize = sizeof(WNDCLASSEXW);
-    wcFrame.lpfnWndProc = FrameWndProc;
+    wcFrame.lpfnWndProc = ChildWndProc;
```

Comment:
I may suggest you get rid of that "ChildWnd" stuff and just use "FrameWnd" in the code, if possible

Reply:
Used the name of `FrameWndProc`.